README.md
# ELS - Enterprise Linux/EuroLinux Sources - glade
 
## Usage:
  Checkout branch or tag. Use `make sources` to recreate  SOURCE0 archive.


src/ChangeLog
# Generated by Makefile. Do not edit.

commit f65173e5070c4710b16553b6dabfc4e6b4a47594
Author: Марко Костић <marko.m.kostic@gmail.com>
Date:   Wed Mar 28 20:40:37 2018 +0000

    Update Serbian translation

 po/sr.po | 338 ++++++++++++++++++++++++++++++++++-----------------------------
 1 file changed, 182 insertions(+), 156 deletions(-)

commit d34d3c9f3f288290843dd8402a7559076ebab2ad
Author: Piotr Drąg <piotrdrag@gmail.com>
Date:   Tue Mar 27 23:13:57 2018 +0200

    Update Polish translation

 help/pl/figures/main-window.png | Bin 39528 -> 39642 bytes
 1 file changed, 0 insertions(+), 0 deletions(-)

commit 7a4b06d5616aab366b75d30334045f249127270a
Author: Guillaume Bernard <associations@guillaume-bernard.fr>
Date:   Mon Mar 26 19:59:11 2018 +0000

    Update French translation

 po/fr.po | 52 ++++++++++++++++++++++++++++++++++++++++++----------
 1 file changed, 42 insertions(+), 10 deletions(-)

commit d14c0d4fe27374cd36622443622ca66e5164aeed
Author: Cheng-Chia Tseng <pswo10680@gmail.com>
Date:   Mon Mar 26 10:15:59 2018 +0000

    Update Chinese (Taiwan) translation

 po/zh_TW.po | 2361 ++++++++++++++++++++++++++++++++---------------------------
 1 file changed, 1263 insertions(+), 1098 deletions(-)

commit 81bbd61fedd0ec0544b25b3ce3a688cda4ff3c46
Author: Andika Triwidada <atriwidada@gnome.org>
Date:   Sun Mar 25 18:50:28 2018 +0000

    Update Indonesian translation

 po/id.po | 270 ++++++++++++---------------------------------------------------
 1 file changed, 50 insertions(+), 220 deletions(-)

commit f09977c773448ff25571a700f8fe8e4c52c24674
Author: Marek Černocký <marek@manet.cz>
Date:   Sun Mar 25 09:16:16 2018 +0200

    Updated Czech translation

 po/cs.po | 46 ++++++++++++++++++++++++++++++++++++----------
 1 file changed, 36 insertions(+), 10 deletions(-)

commit 137fd19bf5940cf73f8654ffd0595128bfa36c99
Author: Juan Pablo Ugarte <juanpablougarte@gmail.com>
Date:   Sat Mar 24 13:25:05 2018 -0300

    GladeGtkHeaderBar: mark title and subtitle as translatable
    
    Fix bug #782753 "Glade does not allow ApplicationWindow's title to be marked as translatable"

 plugins/gtk+/gtk+.xml.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

commit f33da248acc879dff796159be7acb4bd504c2c73
Author: Juan Pablo Ugarte <juanpablougarte@gmail.com>
Date:   Sat Mar 24 13:24:05 2018 -0300

    GladeGtkDialog: chain up post_create() method
    
    GtkWindow adaptor handles use-csd property

 plugins/gtk+/glade-gtk-dialog.c | 3 +++
 1 file changed, 3 insertions(+)

commit 1f7c3e2e594df461117f1c1e136b9923789654d4
Author: Juan Pablo Ugarte <juanpablougarte@gmail.com>
Date:   Sat Mar 24 13:22:48 2018 -0300

    GladeGtkWindow: fix regression in get_children() method
    
    Allways return titlebar even if use-csd is not set

 plugins/gtk+/glade-gtk-window.c | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

commit fc01b4e3f086d58c3557d185c46dc4016a483a32
Author: Piotr Drąg <piotrdrag@gmail.com>
Date:   Fri Mar 23 23:48:17 2018 +0100

    GladeAdaptorChooser: add missing translation domain

 gladeui/glade-adaptor-chooser.ui | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

commit f2dccc0e9ab31884f70b77b4b21ae5613b5f4fd4
Author: Piotr Drąg <piotrdrag@gmail.com>
Date:   Fri Mar 23 23:47:25 2018 +0100

    Update Polish translation

 po/pl.po | 24 ++++++++++++------------
 1 file changed, 12 insertions(+), 12 deletions(-)

commit 5ed1c6070277b6782c4f5773d83877a13018c0ef
Author: Juan Pablo Ugarte <juanpablougarte@gmail.com>
Date:   Thu Mar 22 18:34:24 2018 -0300

    Fix build

 plugins/gtk+/glade-gtk-widget.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

commit 49103b5833c9493472d20d64ab9c0c9c3d6613cc
Author: Juan Pablo Ugarte <juanpablougarte@gmail.com>
Date:   Thu Mar 22 18:14:38 2018 -0300

    GladeGtkWidget: save accessible-role atk property
    
    Fix memory leaks in glade_gtk_widget_write_atk_property() and glade_gtk_widget_write_atk_action()
    Do not serialize atk object id if parent does not have one
    
    Fix bug #794518 "Does not write the accessible-role accessibility property"

 plugins/gtk+/glade-gtk-widget.c | 42 +++++++++++++++++++++++------------------
 1 file changed, 24 insertions(+), 18 deletions(-)

commit b85d9c638809d83fffbe3ef0c184de5415d28293
Author: Fran Dieguez <frandieguez@gnome.org>
Date:   Wed Mar 21 08:48:39 2018 +0000

    Update Galician translation

 po/gl.po | 41 ++++++++++++++++++++++++++++++++++++-----
 1 file changed, 36 insertions(+), 5 deletions(-)

commit 728ebab0e1637b71064c57cc734c297ff2e5f7c8
Author: Juan Pablo Ugarte <juanpablougarte@gmail.com>
Date:   Mon Mar 19 20:44:50 2018 -0300

    GladeGtkWindow: fix regression in csd handling
    
    Fixes bug #794492 "Glade always re-enables the client-side-decoration option"

 plugins/gtk+/glade-gtk-window.c | 32 ++++++++++++++++++++++++++++----
 plugins/gtk+/gtk+.xml.in        |  1 +
 2 files changed, 29 insertions(+), 4 deletions(-)

commit 2d38f33654a12467d7512e50db61907d2ce42e54
Author: Aurimas Černius <aurisc4@gmail.com>
Date:   Mon Mar 19 22:57:37 2018 +0200

    Updated Lithuanian translation

 po/lt.po | 34 +++++++++++++++++++++++++++++++---
 1 file changed, 31 insertions(+), 3 deletions(-)

commit f84a52f983ec7bd891686c1c545bab91952b9c60
Author: Mario Blättermann <mario.blaettermann@gmail.com>
Date:   Mon Mar 19 03:12:14 2018 +0000

    Update German translation

 po/de.po | 257 +++++++++++++++++++++++++++++++++++----------------------------
 1 file changed, 143 insertions(+), 114 deletions(-)

commit bb211dfdeb1883957b79e28220c9359065e8664d
Author: Rafael Fontenelle <rafaelff@gnome.org>
Date:   Sun Mar 18 22:29:20 2018 +0000

    Update Brazilian Portuguese translation

 po/pt_BR.po | 64 ++++++++++++++++++++++++++++++++++---------------------------
 1 file changed, 36 insertions(+), 28 deletions(-)

commit ebce3d60b442924f6b4fd42a49cd4778b1bb86b4
Author: Sebastian Rasmussen <sebras@gmail.com>
Date:   Sun Mar 18 20:24:32 2018 +0000


src/plugins/gtk+/glade-gtk-marshallers.h
/* This file is generated by glib-genmarshal, do not modify it. This code is licensed under the same license as the containing project. Note that it links to GLib, so must comply with the LGPL linking clauses. */
#ifndef __GLADE_GTK_MARSHAL_MARSHAL_H__
#define __GLADE_GTK_MARSHAL_MARSHAL_H__

#include <glib-object.h>

G_BEGIN_DECLS

/* VOID:POINTER,POINTER (./glade-gtk-marshallers.list:1) */
extern
void glade_gtk_marshal_VOID__POINTER_POINTER (GClosure     *closure,
                                              GValue       *return_value,
                                              guint         n_param_values,
                                              const GValue *param_values,
                                              gpointer      invocation_hint,
                                              gpointer      marshal_data);

/* VOID:POINTER (./glade-gtk-marshallers.list:2) */
#define glade_gtk_marshal_VOID__POINTER	g_cclosure_marshal_VOID__POINTER

/* VOID:STRING,ULONG,UINT,STRING (./glade-gtk-marshallers.list:3) */
extern
void glade_gtk_marshal_VOID__STRING_ULONG_UINT_STRING (GClosure     *closure,
                                                       GValue       *return_value,
                                                       guint         n_param_values,
                                                       const GValue *param_values,
                                                       gpointer      invocation_hint,
                                                       gpointer      marshal_data);

/* VOID:OBJECT (./glade-gtk-marshallers.list:4) */
#define glade_gtk_marshal_VOID__OBJECT	g_cclosure_marshal_VOID__OBJECT

/* VOID:STRING (./glade-gtk-marshallers.list:5) */
#define glade_gtk_marshal_VOID__STRING	g_cclosure_marshal_VOID__STRING

/* VOID:INT,INT (./glade-gtk-marshallers.list:6) */
extern
void glade_gtk_marshal_VOID__INT_INT (GClosure     *closure,
                                      GValue       *return_value,
                                      guint         n_param_values,
                                      const GValue *param_values,
                                      gpointer      invocation_hint,
                                      gpointer      marshal_data);

/* VOID:OBJECT,OBJECT (./glade-gtk-marshallers.list:7) */
extern
void glade_gtk_marshal_VOID__OBJECT_OBJECT (GClosure     *closure,
                                            GValue       *return_value,
                                            guint         n_param_values,
                                            const GValue *param_values,
                                            gpointer      invocation_hint,
                                            gpointer      marshal_data);

/* VOID:OBJECT,BOOLEAN (./glade-gtk-marshallers.list:8) */
extern
void glade_gtk_marshal_VOID__OBJECT_BOOLEAN (GClosure     *closure,
                                             GValue       *return_value,
                                             guint         n_param_values,
                                             const GValue *param_values,
                                             gpointer      invocation_hint,
                                             gpointer      marshal_data);

/* VOID:STRING,STRING,STRING (./glade-gtk-marshallers.list:9) */
extern
void glade_gtk_marshal_VOID__STRING_STRING_STRING (GClosure     *closure,
                                                   GValue       *return_value,
                                                   guint         n_param_values,
                                                   const GValue *param_values,
                                                   gpointer      invocation_hint,
                                                   gpointer      marshal_data);

/* OBJECT:POINTER (./glade-gtk-marshallers.list:10) */
extern
void glade_gtk_marshal_OBJECT__POINTER (GClosure     *closure,
                                        GValue       *return_value,
                                        guint         n_param_values,
                                        const GValue *param_values,
                                        gpointer      invocation_hint,
                                        gpointer      marshal_data);

/* OBJECT:OBJECT,UINT (./glade-gtk-marshallers.list:11) */
extern
void glade_gtk_marshal_OBJECT__OBJECT_UINT (GClosure     *closure,
                                            GValue       *return_value,
                                            guint         n_param_values,
                                            const GValue *param_values,
                                            gpointer      invocation_hint,
                                            gpointer      marshal_data);

/* BOOLEAN:STRING (./glade-gtk-marshallers.list:12) */
extern
void glade_gtk_marshal_BOOLEAN__STRING (GClosure     *closure,
                                        GValue       *return_value,
                                        guint         n_param_values,
                                        const GValue *param_values,
                                        gpointer      invocation_hint,
                                        gpointer      marshal_data);

/* BOOLEAN:BOXED (./glade-gtk-marshallers.list:13) */
extern
void glade_gtk_marshal_BOOLEAN__BOXED (GClosure     *closure,
                                       GValue       *return_value,
                                       guint         n_param_values,
                                       const GValue *param_values,
                                       gpointer      invocation_hint,
                                       gpointer      marshal_data);

/* BOOLEAN:OBJECT (./glade-gtk-marshallers.list:14) */
extern
void glade_gtk_marshal_BOOLEAN__OBJECT (GClosure     *closure,
                                        GValue       *return_value,
                                        guint         n_param_values,
                                        const GValue *param_values,
                                        gpointer      invocation_hint,
                                        gpointer      marshal_data);

/* BOOLEAN:OBJECT,BOXED (./glade-gtk-marshallers.list:15) */
extern
void glade_gtk_marshal_BOOLEAN__OBJECT_BOXED (GClosure     *closure,
                                              GValue       *return_value,
                                              guint         n_param_values,
                                              const GValue *param_values,
                                              gpointer      invocation_hint,
                                              gpointer      marshal_data);

/* BOOLEAN:OBJECT,POINTER (./glade-gtk-marshallers.list:16) */
extern
void glade_gtk_marshal_BOOLEAN__OBJECT_POINTER (GClosure     *closure,
                                                GValue       *return_value,
                                                guint         n_param_values,
                                                const GValue *param_values,
                                                gpointer      invocation_hint,
                                                gpointer      marshal_data);

/* BOOLEAN:OBJECT,BOOLEAN (./glade-gtk-marshallers.list:17) */
extern
void glade_gtk_marshal_BOOLEAN__OBJECT_BOOLEAN (GClosure     *closure,
                                                GValue       *return_value,
                                                guint         n_param_values,
                                                const GValue *param_values,
                                                gpointer      invocation_hint,
                                                gpointer      marshal_data);

/* BOOLEAN:OBJECT,UINT (./glade-gtk-marshallers.list:18) */
extern
void glade_gtk_marshal_BOOLEAN__OBJECT_UINT (GClosure     *closure,
                                             GValue       *return_value,
                                             guint         n_param_values,
                                             const GValue *param_values,
                                             gpointer      invocation_hint,
                                             gpointer      marshal_data);

/* BOOLEAN:OBJECT,OBJECT (./glade-gtk-marshallers.list:19) */
extern
void glade_gtk_marshal_BOOLEAN__OBJECT_OBJECT (GClosure     *closure,
                                               GValue       *return_value,
                                               guint         n_param_values,
                                               const GValue *param_values,
                                               gpointer      invocation_hint,
                                               gpointer      marshal_data);

/* BOOLEAN:OBJECT,STRING (./glade-gtk-marshallers.list:20) */
extern
void glade_gtk_marshal_BOOLEAN__OBJECT_STRING (GClosure     *closure,
                                               GValue       *return_value,
                                               guint         n_param_values,
                                               const GValue *param_values,
                                               gpointer      invocation_hint,
                                               gpointer      marshal_data);

/* BOOLEAN:STRING,STRING,STRING,BOXED (./glade-gtk-marshallers.list:21) */
extern
void glade_gtk_marshal_BOOLEAN__STRING_STRING_STRING_BOXED (GClosure     *closure,
                                                            GValue       *return_value,
                                                            guint         n_param_values,
                                                            const GValue *param_values,
                                                            gpointer      invocation_hint,
                                                            gpointer      marshal_data);

/* BOOLEAN:STRING,BOXED,OBJECT (./glade-gtk-marshallers.list:22) */
extern
void glade_gtk_marshal_BOOLEAN__STRING_BOXED_OBJECT (GClosure     *closure,
                                                     GValue       *return_value,
                                                     guint         n_param_values,
                                                     const GValue *param_values,
                                                     gpointer      invocation_hint,
                                                     gpointer      marshal_data);

/* STRING:OBJECT (./glade-gtk-marshallers.list:23) */
extern
void glade_gtk_marshal_STRING__OBJECT (GClosure     *closure,
                                       GValue       *return_value,
                                       guint         n_param_values,
                                       const GValue *param_values,
                                       gpointer      invocation_hint,
                                       gpointer      marshal_data);

/* INT:OBJECT,BOXED (./glade-gtk-marshallers.list:24) */
extern
void glade_gtk_marshal_INT__OBJECT_BOXED (GClosure     *closure,
                                          GValue       *return_value,
                                          guint         n_param_values,
                                          const GValue *param_values,
                                          gpointer      invocation_hint,
                                          gpointer      marshal_data);


G_END_DECLS

#endif /* __GLADE_GTK_MARSHAL_MARSHAL_H__ */
